### ISSUE TITLE
Typescript support

### ISSUE THREAD

**Issue by korifey91** [jarrodconnolly/sequelize-slugify#54]
* **Node.js**: 14.17.5

### What steps will reproduce the bug?

Install `sequelize-slugify` to a typescript project

### What is the expected behavior?

Types are defined

### What do you see instead?

```
src/posts/post.entity.ts:11:30 - error TS2307: Cannot find module 'sequelize-slugify' or its corresponding type declarations.
2021-08-17T08:13:24.667115708Z 
2021-08-17T08:13:24.667126875Z 11 import SequelizeSlugify from 'sequelize-slugify';
2021-08-17T08:13:24.667267208Z                                 ~~~~~~~~~~~~~~~~~~~
```

### Additional information

I found that there are no types in the installed package.
![image](https://user-images.githubusercontent.com/40362351/129691001-619d7628-aabe-4b58-b5b9-fa89d3ab59b1.png)

**Comment by jarrodconnolly** [comment]
Good catch, thank you!

I believe we were just not publishing the types file in the npm package. 

I have released v1.6.0 which now includes the types file referenced correctly in package.json

It appears to be working from a simple test TS project, please let me know if this is what you were expecting.

**Comment by korifey91** [comment]
@jarrodconnolly Yes, thank you!
